### PR TITLE
Aligning Runner and RH claims for v3

### PR DESCRIPTION
**Pull request by pricem14pc**
# Context
As the set of expected claims has changed for launching Census eQ from Census RH, the documentation required updating to reflect this.

### CODE REVIEW

File: docs/respondent_management_to_electronic_questionnaire.rst [hunk 1]
```diff
@@ -1,55 +1,74 @@
...
+``jti``
+   A unique identifier for the JWT
+``tx_id``
+  see :doc:`jwt_profile`
+``response_id``
+  A unique identifier for the questionnaire response
+``questionnaire_id``
+  The Census Questionnaire ID
+``case_type``
+  The type of Census case (e.g. HH, HI, CE, CI)
```
Comment (bitdivision) [inline docs/respondent_management_to_electronic_questionnaire.rst]: This should ideally be a full enumeration

File: docs/respondent_management_to_electronic_questionnaire.rst [hunk 2]
```diff
@@ -1,55 +1,74 @@
...
+``region_code``
+  The Region Code of the questionnaire response
```
Comment (bitdivision) [inline docs/respondent_management_to_electronic_questionnaire.rst]: What format does the region_code take?

File: docs/respondent_management_to_electronic_questionnaire.rst [hunk 3]
```diff
@@ -1,55 +1,74 @@
...
+  The id assigned by the respondent management system (to be removed, hardcoded by RH as 1234567890)
+``period_id``
+  A numerical reference to either a month or quarter time period (hardcoded by RH as 1)
+
+Census Future Requried Fields
+-----------------------------
+The following metadata keys will be introduced in due course
+
+``survey``
+  The survey being launched (e.g census or ccs)
```
Comment (bitdivision) [inline docs/respondent_management_to_electronic_questionnaire.rst]: A full enumeration would be better here

File: docs/conf.py [hunk 4]
```diff
@@ -54,9 +54,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = u'1.0'
+version = u'3.0'
 # The full version, including alpha/beta/rc tags.
-release = u'1.0'
+release = u'3.7.0'
```
Comment (JamesGardiner) [inline docs/conf.py]: Same again as above, the version should be incremented independently of the products using the definition. 
```suggestion
release = u'2.0.0'
```

File: docs/conf.py [hunk 5]
```diff
@@ -54,9 +54,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = u'1.0'
+version = u'3.0'
```
Comment (JamesGardiner) [inline docs/conf.py]:
```suggestion
version = u'2.0'
```

File: docs/conf.py (hunk 4 above)
Comment (pricem14pc) [reply docs/conf.py]: I'll change these. However, this is where the defining this outside of runner actually causes more confusion than adding benefit. Admittedly its not a runner API but this interface is undoubtably bound to a specific version of runner.

File: docs/respondent_management_to_electronic_questionnaire.rst (hunk 2 above)
Comment (pricem14pc) [reply docs/respondent_management_to_electronic_questionnaire.rst]: Added ISO spec

File: docs/conf.py (hunk 5 above)
[the remark this answers is not in the document]
Comment (pricem14pc) [reply docs/conf.py]: Done

File: docs/respondent_management_to_electronic_questionnaire.rst (hunk 1 above)
Comment (pricem14pc) [reply docs/respondent_management_to_electronic_questionnaire.rst]: Done

File: docs/respondent_management_to_electronic_questionnaire.rst (hunk 3 above)
Comment (pricem14pc) [reply docs/respondent_management_to_electronic_questionnaire.rst]: Done

**Comment by bitdivision** [comment]
We should add `schema_name` to these claims. Either (`schema_name` OR (`survey`, `case_type`, `region_code`)) are required. If both of these are passed, then the survey, case_type and region_code will override the schema_name.

schema_name is the name of the schema as per the /schemas endpoint in runner. This corresponds to the filename with no file extension. e.g. `test_checkbox` or `census_individual_gb_eng`

**Comment by pricem14pc** [comment]
> I have also changed `instrument_id` in the downstream payload to `schema_name` corresponding to the input metadata. I don't know whether that should go in here or seperately.
> 
> Given the number of changes on my branch, it may be worth holding off on merging this until everyone is happy with how that works.
> 
> [ONSdigital/eq-survey-runner@v3...eq-2954-change-eq-id-and-form-type](https://github.com/ONSdigital/eq-survey-runner/compare/v3...eq-2954-change-eq-id-and-form-type)

I'll update this definition and the new downstream one to reflect the schema_name change.

File: docs/respondent_management_to_electronic_questionnaire.rst [hunk 6]
```diff
@@ -1,55 +1,75 @@
...
-  The name of the responding unit. Could be a business name or person name. Not required, however at least one of ``ru_name`` and ``trad_as`` **must** be present
-``trad_as``
-  Temporary until wider refactor. Not required, however at least one of ``ru_name`` and ``trad_as`` **must** be present
+  The reporting unit reference id (e.g. UPRN)
+``account_service_url``
+  The url of the account service (e.g. RH) used to launch the survey
+``eq_id``
+  The eQ questionnaire schema id (to be removed, hardcoded by RH as "census")
+``form_type``
+  The particular form_type for a responding unit (to be removed, hardcoded by RH as "individual_gb_eng")
```
Comment (bitdivision) [inline docs/respondent_management_to_electronic_questionnaire.rst]: This should be removed. There's no reference to it in runner anymore, but it won't cause any problems if we receive it. I don't see any reason to keep it in the specification.

File: docs/respondent_management_to_electronic_questionnaire.rst [hunk 7]
```diff
@@ -74,24 +90,22 @@ An example JSON claim
...
-    "roles": [ "role1", "role2" ],
-    "response_id": "QzXMrPqoLiyEyerrED88AbkQoQK0sVVX72ZtVphHr0w="
+    "account_service_url": "http://localhost:9092",
+    "display_address": "ONS, Segensworth Road",
+    "channel": "rh",
+    "survey": "CENSUS",
+    "user_id": "1234567890",
+    "period_id": "1",
+    "schema_name": "census_individual_gb_eng",
+    "form_type": "individual_gb_eng",
```
Comment (bitdivision) [inline docs/respondent_management_to_electronic_questionnaire.rst]: Similarly should be removed

File: docs/respondent_management_to_electronic_questionnaire.rst [hunk 8]
```diff
@@ -1,55 +1,75 @@
...
 ``ru_ref``
-  The responding unit reference id - with checkletter appended.
-``ru_name``
-  The name of the responding unit. Could be a business name or person name. Not required, however at least one of ``ru_name`` and ``trad_as`` **must** be present
-``trad_as``
-  Temporary until wider refactor. Not required, however at least one of ``ru_name`` and ``trad_as`` **must** be present
+  The reporting unit reference id (e.g. UPRN)
+``account_service_url``
+  The url of the account service (e.g. RH) used to launch the survey
+``eq_id``
```
Comment (bitdivision) [inline docs/respondent_management_to_electronic_questionnaire.rst]: Should be removed/

File: docs/respondent_management_to_electronic_questionnaire.rst [hunk 9]
```diff
@@ -1,55 +1,75 @@
...
-  The eQ questionnaire id
+``jti``
+   A unique identifier for the JWT
+``tx_id``
+  see :doc:`jwt_profile`
+``response_id``
+  A unique identifier for the questionnaire response
+``questionnaire_id``
+  The Census Questionnaire ID
+``case_type``
```
Comment (bitdivision) [inline docs/respondent_management_to_electronic_questionnaire.rst]: If schema_type is optional, then  should these fields be optional too? Should probably be a caveat below that one or the other should be passed

File: docs/respondent_management_to_electronic_questionnaire.rst (hunk 6 above)
Comment (pricem14pc) [reply docs/respondent_management_to_electronic_questionnaire.rst]: I've removed these

File: docs/respondent_management_to_electronic_questionnaire.rst (hunk 8 above)
Comment (pricem14pc) [reply docs/respondent_management_to_electronic_questionnaire.rst]: Removed

File: docs/respondent_management_to_electronic_questionnaire.rst (hunk 9 above)
Comment (pricem14pc) [reply docs/respondent_management_to_electronic_questionnaire.rst]: I've added a comment to reflect this

Comment (bitdivision) [reply docs/respondent_management_to_electronic_questionnaire.rst]: Need to push?

Comment (pricem14pc) [reply docs/respondent_management_to_electronic_questionnaire.rst]: Should be there...

**Pull request merged**